zeal-at-point.el

```diff
@@ -130,19 +137,25 @@ the combined docset.")
 
 (defvar zeal-at-point--docset-history nil)
 
-(defvar zeal-at-point-zeal-version
-  (when (executable-find "zeal")
-    (let ((output (with-temp-buffer
-                    (call-process "zeal" nil t nil "--version")
-                    (buffer-string))))
-      (when (string-match "Zeal \\([[:digit:]\\.]+\\)" output)
-        (match-string 1 output))))
-  "The version of zeal installed on the system.")
-
 (unless (fboundp 'setq-local)
   (defmacro setq-local (var val)
     `(set (make-local-variable ',var) ,val)))
 
+(defvar zeal-at-point-zeal-version nil
+  "The version of zeal installed on the system.")
+
+(defun zeal-at-point-get-version ()
+  "Find installed zeal version."
+  (or zeal-at-point-zeal-version
```

Comment:
I suppose you could add another variable to check changes to the exe location.  Seems overkill to me though, the user would need to change the exe location during an emacs session for it to matter right?